src/components/Link.js

```diff
@@ -44,7 +44,7 @@ class CSVLink extends React.Component {
       <a download={filename} {...rest}
          ref={link => (this.link = link)}
          href={this.buildURI(data, uFEFF, headers, separator)}
-         onClick={evt => this.handleLegacy(evt, data, headers, separator, filename)}>
+         onClick={evt => this.handleLegacy(evt, data, headers, separator, filename, uFEFF)}>
```

Comment:
No longer needed with current master

Reply:
@mriccid I can't actually remember the reason for this. Can you fix the merge conflicts and I'll re-review